addons/website/static/src/xml/html_editor.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website.AutoCompleteInLinkpopover" t-inherit="web.AutoComplete">
+        <xpath expr="//input[@t-ref='input']" position="attributes">
```

Comment:
In website, we don't indent the first level of XML files. Just add blank lines around the templates. Thanks :pray:

Reply:
yess, I added the blank lines, but should I also remove the indentation before the `<t>` ? I'm asking to just make sure I understood it, cause I saw some existing templates are with indentation while some are not

Reply:
Yes, that way is the one I recommend (and enforce when I can):

```xml
<?xml version="1.0" encoding="UTF-8"?>
<templates xml:space="preserve">

<t t-name="website.AutoCompleteInLinkpopover" t-inherit="web.AutoComplete">

</t>

</templates>
```

Reply:
Thanks! I will fast change it and repush :pray: